pkg/pb/encryption.proto

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package encryption;
+
+message EncryptionParameters {
+  CipherSuite cipher_suite = 1;
+  int64 block_size = 2;
+}
+
+message CipherSuite {
```

Comment:
Ah also, this probably could just be `enum CipherSuite {` no need for the extra nesting. I think.

Reply:
have another PR going and i can update this